src/cidr-functions/getOccupiedCidrBlocks.js

```diff
@@ -21,14 +22,16 @@ function getRoutes(routeTables) {
 
     routeTables.forEach(routeTable => {
         routeTable.Routes.forEach(route => {
-            uniqueRoutes.add(route.DestinationCidrBlock);
+            console.log('Route', route)
+            if(route.DestinationCidrBlock) {uniqueRoutes.add(route.DestinationCidrBlock)};
```

Comment:
would be good to make this more declarative as to why we are doing this

something like` if (ip4)`